audio_to_text_summarization_using_openai/main.bal

```diff
@@ -1,38 +1,28 @@
 import ballerina/http;
 import ballerina/io;
-import ballerinax/openai.text;
+import ballerinax/openai.chat;
 import ballerinax/openai.audio;
 import ballerinax/twitter;
 
 configurable string openAIToken = ?;
-
-configurable string twitterApiKey = ?;
-configurable string twitterApiSecret = ?;
-configurable string twitterAccessToken = ?;
-configurable string twitterAccessTokenSecret = ?;
+configurable string token = ?;
 
 const string AUDIO_FILE = "test.mp3";
-const string AUDIO_FILE_PATH = "podcast-clips/" + AUDIO_FILE;
+const string AUDIO_FILE_PATH = "./audo_clips/" + AUDIO_FILE;
 const int BINARY_LENGTH = 1000000;
 
 // Twitter API client configuration
-twitter:ConnectionConfig twitterConfig = {
-    apiKey: twitterApiKey,
-    apiSecret: twitterApiSecret,
-    accessToken: twitterAccessToken,
-    accessTokenSecret: twitterAccessTokenSecret
-};
 
 public function main(string podcastURL) returns error? {
     // Creates a HTTP client to download the audio file
     http:Client podcastEP = check new (podcastURL);
-    http:Response httpResp = check podcastEP->/get();
+    http:Response httpResp = check podcastEP->get("");
```

Comment:
whats the reason for this change?

Reply:
> whats the reason for this change?

The reason for this is that it is the only way it is currently supported.
For your reference: https://discord.com/channels/957996897782616114/1286605582022873141/1286607360336334850